projects/mercury/src/dashboard/DashboardPage.js

```diff
@@ -0,0 +1,76 @@
+    const {views} = useContext(MetadataViewContext);
+    const {externalMetadataSources} = useContext(ExternalMetadataSourceContext);
+    const canViewMetadata = currentUser && currentUser.canViewPublicMetadata && views && views.length > 0;
+
+    return (
+        <Grid container justifyContent="center" spacing="5">
+            <BreadCrumbs />
+            <Paper className={classes.mainPage}>
+                <Grid container justifyContent="center" spacing="5">
+                    <Grid container justifyContent="center" spacing="5">
+                        <Typography className={classes.customFont} variant="h3" paragraph>
+                            {APPLICATION_NAME}
+                        </Typography>
+                    </Grid>
+                    <Grid container justifyContent="center" spacing="5" className={classes.header}>
+                        <Typography variant="h5" paragraph>
+                            research data management
+                        </Typography>
+                    </Grid>
+                    <Grid container justifyContent="left" spacing="20" className={classes.textRow}>
+                        <Grid item xs={6} md={5}>
+                            <Typography variant="body1" paragraph>
+                                {APPLICATION_NAME} contains your research metadata. Click on one of the domains and start exploring.
+                            </Typography>
+                        </Grid>
+                        <Grid item xs={6} md={2} />
+                        <Grid item xs={6} md={5}>
+                            <Typography variant="body1" paragraph>
+                                For more details on how to use Fairspace, e.g. how to query the API, please refer the {" "}
+                                <Link href="https://docs.fairway.app/">
+                                    user manual
+                                </Link>.
+                            </Typography>
+                        </Grid>
+                    </Grid>
+                    <Grid container justifyContent="center" spacing="5">
+                        {canViewMetadata && (
+                            <DomainInfo
+                                domainName={METADATA_VIEW_MENU_LABEL}
+                                domainLink="/metadata-views"
+                            />
+                        )}
+                        {canViewMetadata && externalMetadataSources.map((source) => (
+                            <DomainInfo
+                                domainName={source.label}
+                                domainLink={source.name}
+                            />
+                        ))}
+                    </Grid>
+                </Grid>
+            </Paper>
+        </Grid>
+    );
+};
+
+const ContextualDashboardPage = (props) => {
+    const {currentUser} = useContext(UserContext);
+
+    return <DashboardPage currentUser={currentUser} {...props} />;
```

Comment:
not sure why, but somewhere here there is a js warning:
> Warning: Each child in a list should have a unique "key" prop.
Check the render method of `DashboardPage`. See https://reactjs.org/link/warning-keys for more information.
    at WithStyles (http://localhost:3000/static/js/bundle.js:86329:21)
    at DashboardPage (http://localhost:3000/static/js/bundle.js:9466:27)
    at ContextualDashboardPage (http://localhost:3000/static/js/bundle.js:9635:71)
    at WithStyles (http://localhost:3000/static/js/bundle.js:86329:21)
    at Route (http://localhost:3000/static/js/bundle.js:217149:29)
    at Switch (http://localhost:3000/static/js/bundle.js:217351:29)
    at WorkspaceRoutes (http://localhost:3000/static/js/bundle.js:36336:70)